src/api/test/functional/backend_test.rb

```diff
@@ -13,8 +13,8 @@ def test_validate_bsxml
       # map schema names
       if ['about', 'activity', 'added_timestamp', 'announcement', 'announcements', 'new_announcement', 'architecture', 'attrib', 'attrib_type',
           'attrib_namespace', 'attribute_namespace_meta', 'collection_objects_by_tag',
-          'collection_objects_with_tags_by_user', 'configuration', 'directory_filelist', 'directory_view', 'download_counter',
-          'download_counter_summary', 'download_stats', 'group', 'highest_rated', 'issue_tracker',
+          'collection_objects_with_tags_by_user', 'configuration', 'directory_filelist', 'directory_view',
+          'download_stats', 'group', 'highest_rated', 'issue_tracker',
```

Comment:
btw: the download_stats DTD is just as dead.

Reply:
Thanks for the hint!